docs/src/guides/run-ingest-on-terra.rst

```diff
@@ -0,0 +1,79 @@
+**************************
+Run Data Ingest on Terra
+**************************
+
+We have provided two pipelines for importing data into Terra:
+
+* GenBank Ingest - pull a public dataset and send them through our preprocessing scripts.
+* GISAID Ingest - pull a private dataset if a user has their own API key, account, and password.
+
+The pipelines were mainly motivated to provide access to our data pre-processing scripts. Currently, these are focused on pulling datasets for the ncov workflow and is roughly diagramed below:
+
+.. image:: ../images/terra-ingest.png
+
+
+Import ``ingest`` wdl workflow from Dockstore
+=============================================
+
+1. `Setup a Terra account <https://terra.bio/>`_
+#. Navigate to Dockstore: `ncov:wdl/genbank_ingest`_ or `ncov:wdl/gisaid_ingest`_ depending on if you wish to pull open (genbank) data or private (and have an gisaid api key) data.
```

Comment:
```suggestion
#. Navigate to Dockstore: `ncov:wdl/genbank_ingest`_ or `ncov:wdl/gisaid_ingest`_ depending on if you wish to pull open (genbank) data or private (and have an GISAID API key) data.
```

Reply:
To be exact, I would update to `GISAID API endpoint`